ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__arm64-nightly.yaml

```diff
@@ -527,8 +527,8 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test-heterogeneous
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-ipsec
-- as: aws-ipi-private-f14-disasterrecovery
-  cron: 36 19 10,25 * *
+- as: aws-ipi-private-f28-disasterrecovery
```

Comment:
Can you also help update the older version of arm ci jobs on 4.12+

Reply:
@jianlinliu I will open another pr to fix arm dr, and this one is only for amd dr update, is it make sense?

Reply:
okay, if you never care about arm dr jobs, I guess you can directly remove them in another PR.